addons/controllers/testdata/infrastructure_vspherecluster.yaml

```diff
@@ -0,0 +1,1723 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
Looks like this is external CRD. You can try to add additional CRDs in https://github.com/vmware-tanzu/tanzu-framework/blob/main/addons/controllers/suite_test.go#L92, so you don't need to port in the whole file.

Reply:
that's a good idea. I've removed the CR file and it works.